pages.fr/common/git-clone.md

```diff
@@ -1,24 +1,36 @@
 # git clone
 
-> Clone un dépôt existant.
+> Cloner un dépôt existant.
```

Comment:
Please use présent de l'indicatif.

Reply:
Hey,
Thank you, I'm new to GitHub and I do not totally understand the + and - thing in file changes (like in wich file is the + line added and in wich file was the - line removed).
So, in this case, did you actually modified the `git-clone.md` file or not ? 'cause @blueskyson did if I understand it correctly (since I commited his suggestion), but in this case you're not suggesting a commit modification but rather just placing a comment in the file, right ?

Reply:
well, in this case you modified an already existing line, for Git the old line is a deletion and your version is an addition to the file. Yeah, I commented a file but my comment applies to all the examples.

Reply:
As for how to make suggestions in a pull request, you can take a look at [git docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request). As for write access to a pull request, you granted maintainers permissions to edit when you created a pull request, so we can make suggestions or commit to your branch before merging. See also [git docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

Reply:
Thank you for your replies, I guess i will more often have a look in the git/github docs, should be a good idea

Reply:
@Stijn-Bch do you need any help with your PR?

Reply:
Hi, well I haven't done anything since the last time. I thought I just had to wait for the last review so my PR could be merged :)
But I realize, should I have changed all the verbs to indicatif présent ?